cloud-service-providers/aws/eks/README.md

```diff
@@ -29,6 +29,10 @@ Note: Ensure that you are in the cloud-service-providers/aws/eks directory for t
 
 ## Cluster setup for inference:
 
+You can either use the NIM Helm Chart or NIM Operator. If you would like to use the NIM Operator, please see the instructions [here](nim-operator-setup.md). 
```

Comment:
Try to use impersonal tense avoiding words like "you" in the technical documents.